README.md

```diff
@@ -28,7 +28,7 @@ This is a provider to create [Azure Redhat Openshift](https://docs.microsoft.com
 ### Give Red Hat Openshift Resource Provider network contributor role of Azure network
 
 ```bash
-OPENSHIFT_RP_OBJECT_ID=$(az ad sp list --filter "displayname eq 'Azure Red Hat OpenShift RP'" --query "[?appDisplayName=='Azure Red Hat OpenShift RP'].objectId" --only-show-errors --output tsv)
+OPENSHIFT_RP_OBJECT_ID=$(az ad sp list --filter "displayname eq 'Azure Red Hat OpenShift RP'" --query "[?appDisplayName=='Azure Red Hat OpenShift RP'].id" --only-show-errors --output tsv)
```

Comment:
@kmcolli, which AZ CLI version are you using?  Mine uses objectId instead of id
{
  "azure-cli": "2.36.0",
  "azure-cli-core": "2.36.0",
  "azure-cli-telemetry": "1.0.6",